scripts/explain_manifest/fixtures/alpine-amd64.txt

```diff
@@ -31,11 +31,6 @@
   Needed    :
   - libc.so
 
-- Path      : /usr/local/lib/lua/5.1/cjson.so
```

Comment:
Note to reviewers: this is only removing the _duplicate_ cjson.so that was added in 790a18e1d8. The OpenResty-provided cjson.so remains.